desktop/components/artwork_item/templates/artwork.jade

```diff
@@ -4,7 +4,7 @@
 - displayPurchase = displayPurchase || false
 - showBlurbs = showBlurbs || false
 - isAuction = isAuction || (set && set.get('type') === 'auction artworks') || false
-- hasInstitutionPartner = artwork.get('partner') && artwork.get('partner').type === 'Institution'
+- hasInstitutionPartner = artwork.get('partner') && artwork.get('partner').type === 'Institution' && !artwork.get('forsale')
```

Comment:
Sorry for the slowness. I am not aware of the history of hiding availability for institutional seller works, either. However, I am under the impression that we treat institutional sellers as galleries, instead of institutions, for commercial stuff. That said, this check might not be want we want, as @mzikherman pointed out.

I am kind of confused by the `saleMessage()` and `availabilityMessage()` in the artwork model. Even if we removed this [commit](https://github.com/artsy/force/commit/0b2c0d11cce9b3f4917ef7cd7b5c8c8d0d760837), it will still early return from the following line?